catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -135,6 +138,10 @@
 
   protected boolean shouldConvertToBBox;
 
+  protected int numMultiPointRadiusVertices;
+
+  protected int distanceTolerance;
```

Comment:
❓ Any reason these were made protected instead of private?

Reply:
Since they have public getters these should be private

Reply:
Just following the pattern I noticed in the other variables. I assume it is done that way for inheritance purposes in downstream projects.

Reply:
Right. I made these private so that downstream projects could extend this class and access these variables.